### DISCUSSION TITLE
Allow user to expand GraphiQL UI frame

### DISCUSSION THREAD

**Discussion by haaah** [hasura/graphql-engine#8811] [General] [answered]
Would like to suggest to the team to allow user to expand the current GraphiQL UI Frame. Currently there are a lot of wasted white space underneath.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/8482031/185340883-e7da0574-e9d7-472c-91b7-e7e778666b38.png">

**Comment by meetzaveri** [comment]
Hello,

Thanks for reporting the issue. We have acknowledged this request, and it is being tracked here on this github issue - https://github.com/hasura/graphql-engine/issues/8826

**Comment by rikinsk** [comment, marked as answer]
Tracking as issue [8880](https://github.com/hasura/graphql-engine/issues/8880) instead
